File: Iniciante/1047 - Tempo de Jogo com Minutos/C#/1047 - Tempo de Jogo com Minutos.cs
using System;

class URI {
    static void Main() {
        // recebe os valores na mesma linha
        string[] valor = Console.ReadLine().Split(' ');
            // converte cada variável para int
            int hora_ini = int.Parse(valor[0]);
            int min_ini = int.Parse(valor[1]);
            int hora_fin = int.Parse(valor[2]);
            int min_fin = int.Parse(valor[3]);

        // converte o tempo para minutos
        int tempo = ((hora_fin*60) + min_fin) - ((hora_ini*60) + min_ini);
        
        // caso o tempo seja negativo ou igual a 0
        if(tempo <= 0) {
            tempo = tempo + 1440; // 24 horas == 1440 minutos
        }

        // saída
        Console.WriteLine("O JOGO DUROU {0} HORA(S) E {1} MINUTO(S)", tempo/60, tempo%60);
    }
}

File: Iniciante/1047 - Tempo de Jogo com Minutos/Python/1047 - Tempo de Jogo com Minutos.py
# recebe os valores na mesma linha
hora_ini, min_ini, hora_fin, min_fin = input().split(' ')
# converte cada valor para int
hora_ini, min_ini, hora_fin, min_fin = int(hora_ini), int(min_ini), int(hora_fin), int(min_fin)

# converte o tempo total em minutos
tempo = ((hora_fin*60)+min_fin) - ((hora_ini*60)+min_ini)

# caso o tempo seja negativo ou igual a 0
if tempo <= 0:
    tempo = tempo + 1440 # 24h == 1440 minutos

# saída
print("O JOGO DUROU", int(tempo/60), "HORA(S) E", int(tempo%60), "MINUTO(S)")